JSDemos/Demos/Charts/Colorization/description.md

```diff
@@ -1 +1,14 @@
-The TreeMap component provides a&nbsp;flexible system of&nbsp;tile colorization. You can assign colors to&nbsp;tiles depending on&nbsp;their values or&nbsp;the group they belong&nbsp;to. If&nbsp;colors are specified in&nbsp;your data source, you can use them as&nbsp;well. Finally, the TreeMap includes nine built-in palettes that ensure the superb appearance.
\ No newline at end of file
+This demo shows techniques that you can use to colorize TreeMap tiles. To see each technique in action, use a drop-down editor below the component.
+
+To colorize the TreeMap tiles, you can use the [colorizer](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/) object. This object allows you to specify the colorization [type](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#type):
+
+- *"discrete"* (default)    
+This colorization algorithm paints each tile within a group in a color taken from the [palette](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#palette). When there are multiple groups, colorization begins from scratch in each group. Alternatively, you can force the UI component to continue colorization across groups. For this purpose, set the [colorizeGroups](//Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#colorizeGroups) property to `true`. It colorizes not each tile, but each group of tiles in a single color taken from the palette. This demo shows these two techniques in **Discrete** and **Grouped** modes.
+
+- *"gradient"*    
+If you choose this algorithm, the palette should contain only two colors that the component uses to colorize the smallest and the largest tile respectively. The other tiles will have a tint of either the first or the second color depending on their size. In order to use the *"gradient"* algorithm, you need to set the [range](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#range) property to an array of strictly two values.
+
+- *"range"*    
+This algorithm is similar to *"gradient"*, but the array assigned to the [range](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#range) property should contain more than two values. This array specifies the ranges between each value. The palette of two colors will generate multiple tints - one for each range.
```

Comment:
1. ...`should contain more than two values.`

For my understanding, how many ranges should this property contain?

2. `The palette of two colors will generate multiple tints - one for each range.`

Just for my understanding, if the property contains more than two colors, then why does this sentence mention a "palette of two colors"?

Reply:
1. It depends on the number of items, if you have like 3 of them in each group, than you don't need more than 3 ranges. If you have more, go for more.
2. I'm talking about range in the previous sentence, not a palette of colors. I wrote the last sentence because of two reasons: 1) show a parallel between the gradient mode and the range mode 2) the demo shows a palette of two colors in this mode

Reply:
Okay, thank you for the clarification.